### PR TITLE
Mid December Polaris Sync

### PR DESCRIPTION
**Pull request by Cameron653**
There was 28 stock .dmi file changed that required manual edits, along with 28 conflicting .dm files.

All of which have been fixed! Woo.

Code compiles. This has been long overdue.

### CODE REVIEW

**Comment by Cameron653** [comment]
Ack, let me fix.

**Comment by Cameron653** [comment]
I have no idea. They were conflicting on our side with the new ones, so either a prior sync that touched those files caused it to conflict or something was bugged. The Polaris .dmi files were put there instead. They just had more shading on the doors.

**Comment by Arokha** [comment]
Should probably include a changelog so that when moved into release it doesn't sit and wait there for someone to make one from scratch and they can copypaste this one and others together.

**Comment by Cameron653** [comment]
I really don't have much of an idea of what has been changed in here. I did a changelog a long while back and it was absolutely miserable putting it together.

Someone else can make a changelog for it, since I'd rather not.

**Comment by Leshana** [comment]
Okay, DMI changes look like they are in the correct commit, should be good.
There was a second commit with changes too, but that has been squashed into the merge commit also, this should work now.

**Comment by Cameron653** [comment]
Someone skimmed through the code and sent me this:


- Adds the box cutter knife and teapot?
- Advanced medkit healing items now only heal 3 from 5.
- The HUD has been changed in some way?
- Laser Carbine renamed to the Laser Rifle, and now has a fire delay of 8 in lethal and 5 in suppressive.
- Maint drones can now ventcrawl.
- Adds snowballs. Yay christmas fun!
- Makes game ticks actually work when they're supposed to more often?
- Removes references to clothing being military from the loadout
- Adds huge amounts of new drinks
- Replaces cola machine sprite?
- Fixes rollers and mediborgs?
- Adds metamorphic drink glasses, basically just adding the old style glasses code without removing anything
- Updates prisoner.dm?
- ABS and sleepers reject non-human mobs
- Makes galoshes shockproof.
- More types of sheets can be ground up in a grinder.
- Fixes a few research typos
- Adds (terminator) crawling to the game, working like such:

Mobs now have the ability to "crawl" to an adjacent turf by click-dragging themselves to it. This would allow people unable to stand to still move around, though very slowly. You could also do this to other movable mobs or objects, if you really wanted to.

Moving this way takes about 2.5 seconds, plus half a second for each point of weakness you have. For a person missing a leg, this would take 5 seconds. It only works if the mob doing this is alive, conscious, unrestrained, not stunned or paralyzed, and in an area with gravity - no crawling through space.

Mobs lying on the ground - again, with similar restrictions - are now able to buckle themselves to chairs or beds. For people missing legs or feet, this means no more being stuck helpless on the ground forever because Baldie McGreytide clicked on your wheelchair.

**Comment by SingoDingo** [comment]
You also missed the first two parts of that changelog:
(Things with a "?" next to them may not be 100% accurate due to PRs being weirdly commented/explained.)
- Shotgun clips now only hold 2 from 4
- Makes walls more uniform, especially r-walls?
- Adds hunting rounds that do more damage to mobs and less damage to humanoids
- Adjusts the amount of camera locations the AI can store from 10 to 30.
- Lowers the volume of mouse squeaks?
- Adds some kind of ballistic transportation pod?
- Alters the time it takes to applies cuffs?
- Adds bolas, which are fire-able legcuffs?
- Adds new spider types
- Hacked borgs can no longer be locked down.
- Fixes a few vendor chat log things and uses unused sprites for vendors
- Removes the mosin-nagant, moves the sprites to other guns?
- Fixes how intents interact with drinks, allowing you to splash your drink on someone in harm intent
- Adds exploration equipment including a hooded armour suit with a hood light (for miners?) and a survival bootknife
- Replaces xenoarch GPS with budget /tg/ GPS that is more useful, as it can track other GPSes.
- Adds a bunch of new signs
- Adds the 'pagan', 'orthodox', and 'torah' chaplain bibles, and removes the 'creeper' chaplain bible.
- Adds a bunch of new POIs?
- Lowers block chance on alien armour from 25% to 20%
- Blizzards are about 10C less cold.
- Adds a cap to breathloss, making it cause less oxyloss and only performing the *gasp emote every 10 ticks?
- Winter boots now no longer cover the legs, only the feet. Winter coats now cover the legs to compensate.
- Allows communicators to be made by RnD, shows ghosts communicator IMs
- Mobs can't push objects that they can't pull.
- Removes gendered pronouns on cleaning and floorbots, as well as makes them 'boop' instead of 'beep boop'.
- Adds an advanced roller bed, that takes up less bag space when folded and can be used as a surgery table? Is developed by science, costs phoron.
- Adds advanced health analyzer, that can tell the specific location of bone fractures and internal bleeding, as well as the quantity of medical reagents in the stomach. Is developed by science, costs diamonds.
- Adds a bunch of loadout optimisations, and adds new loadout items, including: A stethoscope, a locket, plain goggles, prescription science goggles(?), science aviators (and prescription variant), a white cane, a colorable cap, a powdered wig, rabbit ears, a cheap lighter, a greatcoat, and a camera.
- Parrots now know galactic common by default? Removes "BAWWWWK george mellons griffing me" from their speaklist.
- Contaminated items now respect gloves
- Clotting crates now cost 100 cargo points from 40.
- Can no longer run around while in a roller chair.
- Adds Reach Variable to Weapons, currently only allowing spears to attack from 2 tiles (and over chairs/other people), making meatshield gladius greytide possible.
- Fixes and/or adds the chainsaw. Let the texas chainsaw massacre begin.
- Adds parachutes.
- Psychiatrists can now spawn with medical HUDs from the loadout.

**Comment by Cameron653** [comment]
Oh! No idea how that happened. Whoops.

File: code/datums/autolathe/arms.dm [hunk 1]
```diff
@@ -147,17 +147,27 @@
...
+/datum/category_item/autolathe/arms/rifle_545_hunter
+	name = "rifle magazine (5.45mm hunting)"
+	path =/obj/item/ammo_magazine/m545/hunter
+
 /datum/category_item/autolathe/arms/machinegun_545
 	name = "machinegun box magazine (5.45)"
 	path =/obj/item/ammo_magazine/m545saw
 	hidden = 1
 
+/datum/category_item/autolathe/arms/machinegun_545_hunter
```
Comment (SpadesNeil) [inline code/datums/autolathe/arms.dm]: Hunting ammunition is stupid. I want it removed.

File: code/datums/supplypacks/contraband.dm [hunk 2]
```diff
@@ -43,10 +43,10 @@
 	containername = "Moghes imports crate"
 	contraband = 1
 
-/datum/supply_packs/security/bolt_rifles_mosin
+/datum/supply_packs/security/bolt_rifles_militia
  	name = "Surplus militia rifles"
  	contains = list(
- 			/obj/item/weapon/gun/projectile/shotgun/pump/rifle/mosin = 3,
+ 			/obj/item/weapon/gun/projectile/shotgun/pump/rifle = 3,
```
Comment (SpadesNeil) [inline code/datums/supplypacks/contraband.dm]: **CHANGE IT BACK**

File: code/game/objects/items/weapons/storage/boxes.dm [hunk 3]
```diff
@@ -395,6 +395,17 @@
 	new /obj/item/weapon/implanter(src)
 	new /obj/item/weapon/implantpad(src)
 
+/obj/item/weapon/storage/box/camerabug
```
Comment (SpadesNeil) [inline code/game/objects/items/weapons/storage/boxes.dm]: Need to review the rest later. This is where I left off.

**Comment by Arokha** [comment]
If you're doing his things above, make sure to put //VOREStation Edit comments.

File: code/datums/supplypacks/contraband.dm (hunk 2 above)
Comment (SpadesNeil) [reply code/datums/supplypacks/contraband.dm]: Override this in contraband_vr.dm

**Comment by Cameron653** [comment]
Aces changes taken care of.

**Comment by Cameron653** [comment]
...Let me fix

**Comment by Cameron653** [comment]
Oh. The mosin was removed from the game.

That'll need to be added later, but not in this PR.

**Comment by Arokha** [comment]
Agreed. None of these changes are game-breaking or anything and can be fixed later. Oh noooo there's a type of ammo in the game that's not even on our map that we need to remove asap. \*cough\*

File: code/datums/autolathe/arms.dm [hunk 4]
```diff
@@ -147,9 +147,9 @@
 	name = "rifle magazine (5.45mm practice)"
 	path =/obj/item/ammo_magazine/m545/practice
 
-/datum/category_item/autolathe/arms/rifle_545_hunter
+/*/datum/category_item/autolathe/arms/rifle_545_hunter //VOREStation Edit Start. By request of Ace
 	name = "rifle magazine (5.45mm hunting)"
-	path =/obj/item/ammo_magazine/m545/hunter
+	path =/obj/item/ammo_magazine/m545/hunter*/ //VOREStation Edit End.
```
Comment (SpadesNeil) [inline code/datums/autolathe/arms.dm]: That's not all of it. CTRL+H the word "_hunter" and remove **all** mentions of hunting ammo. It's all not wanted. It's a dumb idea.

File: code/datums/supplypacks/contraband.dm [hunk 5]
```diff
@@ -46,7 +46,7 @@
 /datum/supply_packs/security/bolt_rifles_militia
  	name = "Surplus militia rifles"
  	contains = list(
- 			/obj/item/weapon/gun/projectile/shotgun/pump/rifle = 3,
+ 			/obj/item/weapon/gun/projectile/shotgun/pump/rifle/mosin = 3, //VOREStation Edit. By request of Ace.
```
Comment (SpadesNeil) [inline code/datums/supplypacks/contraband.dm]: nooooo just do contraband_vr.dm instead

Comment (Cameron653) [reply code/datums/supplypacks/contraband.dm]: I can't. The gun has been deleted from the game. The path doesn't exist any longer.

Comment (SpadesNeil) [reply code/datums/supplypacks/contraband.dm]: nevermind this entirely, you saw my Github comments

**Comment by Cameron653** [comment]
All hunter ammo commented out in arms.dm

**Pull request merged**